distribution/build.gradle

```diff
@@ -629,22 +629,6 @@ subprojects {
   }
 }
 
-['archives:windows-zip',
```

Comment:
We don't use it and its basically just some rewiring of configurations adding nothing but clutter to the build that can be avoided.